locales/en.json

```diff
@@ -1552,6 +1555,12 @@
       "public": "Public Mint"
     }
   },
+  "partyModal": {
+    "title": "Experience The Drop Party",
+    "text": "See and join other collectors browsing the drop in real time.",
+    "no": "OPT OUT",
+    "yes": "DIVE IN!"
```

Comment:
match the design pls

![CleanShot 2024-04-08 at 10 31 11@2x](https://github.com/kodadot/nft-gallery/assets/44554284/2e1f6b36-7b3f-4641-a5f5-435d066f0067)